BT-Tracking/Interface/Data List/DataListVC.swift

```diff
@@ -107,7 +107,7 @@ final class DataListVC: UIViewController, UITableViewDelegate {
     }
 
     private func sendReport() {
-        guard (AppSettings.lastUploadDate ?? Date.distantPast) + (15 * 60) < Date() else {
+        guard (AppSettings.lastUploadDate ?? Date.distantPast) + RemoteValues.uploadWaitingMinutes < Date() else {
```

Comment:
Minutes or seconds?

Reply:
Timeinterval is in seconds, so 15 * 60, is 15 minutes

Reply:
The type returned from RemoteValues is Timeinterval so it's altery in seconds